File: app/src/main/java/cz/kutner/comicsdb/connector/service/NewsService.java
package cz.kutner.comicsdb.connector.service;

import java.util.List;

import cz.kutner.comicsdb.model.NewsItem;
import retrofit.http.GET;
import rx.Observable;

public interface NewsService {
    @GET("/index.php")
    Observable<List<NewsItem>> listNews();
}

File: app/src/main/kotlin/cz/kutner/comicsdb/connector/NewsService.kt
package cz.kutner.comicsdb.connector

import cz.kutner.comicsdb.model.NewsItem
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface NewsService {
    @GET("/api.php?get=news")
    fun listNews(@Query("start") start: Int, @Query("records") records: Int): Call<List<NewsItem>>
}
